lib/JsonpMainTemplatePlugin.js

```diff
@@ -121,12 +121,28 @@ class JsonpMainTemplatePlugin {
 		mainTemplate.plugin("bootstrap", function(source, chunk, hash) {
 			if(chunk.chunks.length > 0) {
 				var jsonpFunction = this.outputOptions.jsonpFunction;
+				var topLevelAwaitHanding = this.topLevelAwaitInChildren(chunk);
+				var finalizeModulesCode = [
+					"while(resolves.length) {",
+					this.indent("resolves.shift()();"),
+					"}",
+					this.entryPointInChildren(chunk) ? [
+						"if(executeModules) {",
+						this.indent([
+							"for(i=0; i < executeModules.length; i++) {",
```

Comment:
This code was moved from https://github.com/webpack/webpack/pull/4707/files/a0cd9aace98e148672c2fe61e328b08b12b86e7b#diff-82578d379e84f28902072d7b5efc5be0L157 -- can be pre-existing